src/pyqt_tools/__main__.py

```diff
@@ -0,0 +1,19 @@
+""" Enables support for calling entrypoints using `python -m` or `pythonw -m`.
+On windows using `pythonw -m pyqt6_tools designer` in a shortcut allows opening
+QtDesigner without showing a command prompt window that stays open the entire
+time that QtDesigner is open.
+"""
+
+
+import importlib
+import pathlib
+import sys
+
+if __name__ == "__main__":
```

Comment:
I would personally drop this since there's no point in importing this for any other purpose and modules that you can both run and import can result in really goofy corner cases when you do happen to do both of those things in one process.

Reply:
True, it doesn't make sense for a `__main__.py` file and leads to the issues in the other note, removed.